osgp/pom.xml

```diff
@@ -6,6 +6,13 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <parent>
+    <groupId>org.opensmartgridplatform</groupId>
+    <artifactId>super-osgp</artifactId>
+    <version>5.15.0-SNAPSHOT</version>
```

Comment:
Now that the version is specified for the parent, can (should?) it not be removed from this POM (line 18)?
I was looking for the groupId as another candidate to be removed as it would match the groupId from the parent POM.
Then I noticed in this POM the groupId is 'org.osgp'. Is this intentionally different from the groupId used elsewhere in the platform?